service/handlers.go

```diff
@@ -221,3 +243,7 @@ func (rh *RouteHandler) isUniqueName(webhookName string, projectID string, apiCl
 	}
 	return 200, nil
 }
+
+func getRoles(r *http.Request) string {
+	return r.Header.Get(RoleAPIHeader)
```

Comment:
The value can potentially be a comma separeted list of roles. like
```
readonly,restrcited
```
Here's the code that sets the header:
```
 Policy policy = ApiUtils.getPolicy();
        if (policy != null) {
            projectHeader = ApiContext.getContext().getIdFormatter()
                    .formatId(AccountConstants.TYPE, Long.toString(policy.getAccountId()))
                    .toString();
            roles = policy.getRoles();
            roleString = StringUtils.join(roles, ",");
        }
        temp.setHeader(ProjectConstants.PROJECT_HEADER, projectHeader);
        temp.setHeader(ProjectConstants.ROLES_HEADER, roleString);
```
I dont know what scenario would lead to having more than one role, but we should attempt to account for that scenario.

The question is, what if the value is like `readonly,member`. What does that even mean? Do you go with they cannot see it because they have the roel readonly or that they can see it bc they have the role member. I guess the right thing to do would be to allow them to see it. But then you would need a list of roles that _are_ allowed to see it.

Maybe you could reproduce that scenario by adding a user to a project as both a member and readonly? But its weird to even do that.

Here is what I'm thinking. Let me know if you have a better idea:
1. Split the header value on commas so that we at least account for more than one role being present.
2. only hide the url if all the values from the split are in your readonlyRoles map

Reply:
after further discussion and investigation. it is effectively impossible to have more than one role in this header. no need to change